wiki/markdown/configuration.md

```diff
@@ -0,0 +1,61 @@
+# Configuration
+
+To be able to execute and use Eiffel-Intelligence Front-end, some configuration need to provided.
+All configurations in made via application.properties file. These properties can also be provided as Java properties arguments as well.
```

Comment:
Should we maybe have examples of a command with some Java properties following this sentence?

Reply:
There is some examples of how to do that on Running Eiffel Intelligence Frontend page.